assets/css/links.css

```diff
@@ -0,0 +1,12 @@
+.grid {
```

Comment:
I think pico already uses a `.grid` style. Maybe you would want to make it a bit more specific, for example `.link-grid` (or reuse some existing style? I don't know if we have any that fit your use case)

Reply:
Ok changed to link-grid, good one thanks